src/reporter/delete.py

```diff
@@ -33,17 +33,24 @@ def delete_entity(entity_id, type_=None, from_date=None, to_date=None):
 
 def delete_entities(entity_type, from_date=None, to_date=None,
                     drop_table=False):
-    with translator_for(fiware_s()) as trans:
-        if drop_table:
-            trans.drop_table(etype=entity_type, fiware_service=fiware_s())
-            logging.getLogger(__name__).info("dropped entity_type {}".format(entity_type))
-            return 'entity table dropped', 204
-
-        deleted = trans.delete_entities(etype=entity_type,
-                                        from_date=from_date,
-                                        to_date=to_date,
-                                        fiware_service=fiware_s(),
-                                        fiware_servicepath=fiware_sp(),)
+        with translator_for(fiware_s()) as trans:
+            if fiware_s() is not None:
+                e = None
+                e = fiware_s().split(",")
+                for i in e:
+                    count = len(e)
+                    if count > 1:
```

Comment:
we could also factor this out in a separate function, e.g.

    has_many_service_paths() -> bool:
       if fiware_sp()...

just a suggestion, we can leave it the way it is now if you don't have time for it.